src/components/Column/ColumnSettings.tsx

```diff
@@ -36,109 +41,89 @@ export const ColumnSettings: FC<ColumnSettingsProps> = ({id, name, color, visibl
     dispatch(Actions.createColumnOptimistically({id: TEMPORARY_COLUMN_ID, name: "", color: randomColor, visible: false, index: columnIndex}));
```

Comment:
this code isn't by you, but still wanted to comment that this actually doesn't lead to a fair distribution of colors, since 100 isn't divisible by 7.
```ts
const randomColor = getColorForIndex(Math.floor(Math.random() * COLOR_ORDER.length));
```
would probably be better.

Reply:
sure, we're all for color equality here:D